lib/scope.go

```diff
@@ -104,18 +118,27 @@ func (s *scope) ParseAndResolveRefWithWorkingDir(ctx context.Context, refstr, so
 	return s.inst.ParseAndResolveRefWithWorkingDir(ctx, refstr, source)
 }
 
-// LoadDataset loads a dataset
-func (s *scope) LoadDataset(ctx context.Context, ref dsref.Ref, source string) (*dataset.Dataset, error) {
-	return s.inst.LoadDataset(ctx, ref, source)
-}
-
-// Loader returns the default dataset ref loader
+// ParseResolveFunc returns the default dataset ref loader
 func (s *scope) ParseResolveFunc() dsref.ParseResolveLoad {
 	return NewParseResolveLoadFunc("", s.inst.defaultResolver(), s.inst)
 }
 
-// GetVersionInfoShim is in the process of being removed. Try not to add new callers.
-func (s *scope) GetVersionInfoShim(ref dsref.Ref) (*dsref.VersionInfo, error) {
-	r := s.inst.Repo()
-	return repo.GetVersionInfoShim(r, ref)
+// Profiles accesses the profile store
+func (s *scope) Profiles() profile.Store {
+	return s.inst.profiles
+}
+
+// Repo returns the repo store
+func (s *scope) Repo() repo.Repo {
+	return s.inst.repo
+}
+
+// Stats returns the stats service
+func (s *scope) Stats() *stats.Service {
+	return s.inst.stats
+}
+
+// Transform returns the transform service
+func (s *scope) Transform() *transform.Service {
+	return s.inst.transform
```

Comment:
This has changed due to https://github.com/qri-io/qri/pull/1691. The `TransformService` no longer exists on `lib`.